src/world/area_kzn/kzn_04/kzn_04_3_npc.c

```diff
@@ -0,0 +1,278 @@
+    EVT_CALL(N(FireBarAI_Main), EVT_PTR(N(AISettings_FireBar_03)))
+    EVT_RETURN
+    EVT_END
+};
+
+NpcSettings N(NpcSettings_FireBar_01) = {
+    .defaultAnim = ANIM_Fire_Brighest_Burn,
+    .height = 12,
+    .radius = 20,
+    .ai = &N(EVS_NpcAI_FireBar_01),
+    .flags = ENEMY_FLAGS_100 | ENEMY_FLAGS_200 | ENEMY_FLAGS_400 | ENEMY_FLAGS_800,
+};
+
+NpcSettings N(NpcSettings_FireBar_02) = {
+    .defaultAnim = ANIM_Fire_Brighest_Burn,
+    .height = 12,
+    .radius = 20,
+    .ai = &N(EVS_NpcAI_FireBar_02),
+    .flags = ENEMY_FLAGS_100 | ENEMY_FLAGS_200 | ENEMY_FLAGS_400 | ENEMY_FLAGS_800,
+};
+
+NpcSettings N(NpcSettings_FireBar_03) = {
+    .defaultAnim = ANIM_Fire_Brighest_Burn,
+    .height = 12,
+    .radius = 20,
+    .ai = &N(EVS_NpcAI_FireBar_03),
+    .flags = ENEMY_FLAGS_100 | ENEMY_FLAGS_200 | ENEMY_FLAGS_400 | ENEMY_FLAGS_800,
+};
+
+NpcSettings N(NpcSettings_FireBar_Extra) = {
+    .defaultAnim = ANIM_Fire_Brighest_Burn,
+    .height = 12,
+    .radius = 25,
+    .flags = ENEMY_FLAGS_100 | ENEMY_FLAGS_200 | ENEMY_FLAGS_400 | ENEMY_FLAGS_800,
+};
+
+StaticNpc N(NpcData_FireBar_01)[] = {
+    {
+        .id = NPC_FireBar_1A,
+        .settings = &N(NpcSettings_FireBar_01),
+        .pos = { 0.0f, 0.0f, 0.0f },
+        .yaw = 0,
+        .flags = NPC_FLAG_PASSIVE,
+        .init = (void*) 0x00004003,
+        .animations = {
+        },
+    },
+    {
+        .id = NPC_FireBar_1B,
+        .settings = &N(NpcSettings_FireBar_Extra),
+        .pos = { 0.0f, 0.0f, 0.0f },
+        .yaw = 0,
+        .flags = NPC_FLAG_PASSIVE,
+        .init = (void*) 0x00004003,
+        .animations = {
+        },
+    },
+    {
+        .id = NPC_FireBar_1C,
+        .settings = &N(NpcSettings_FireBar_Extra),
+        .pos = { 0.0f, 0.0f, 0.0f },
+        .yaw = 0,
+        .flags = NPC_FLAG_PASSIVE,
+        .init = (void*) 0x00004003,
+        .animations = {
+        },
+    },
+    {
+        .id = NPC_FireBar_1D,
+        .settings = &N(NpcSettings_FireBar_Extra),
+        .pos = { 0.0f, 0.0f, 0.0f },
+        .yaw = 0,
+        .flags = NPC_FLAG_PASSIVE,
+        .init = (void*) 0x00004003,
+        .animations = {
+        },
+    },
+};
+
+StaticNpc N(NpcData_FireBar_02)[] = {
+    {
+        .id = NPC_FireBar_2A,
+        .settings = &N(NpcSettings_FireBar_02),
+        .pos = { 0.0f, 0.0f, 0.0f },
+        .yaw = 0,
+        .flags = NPC_FLAG_PASSIVE,
+        .init = (void*) 0x00004003,
+        .animations = {
+        },
+    },
+    {
+        .id = NPC_FireBar_2B,
+        .settings = &N(NpcSettings_FireBar_Extra),
+        .pos = { 0.0f, 0.0f, 0.0f },
+        .yaw = 0,
+        .flags = NPC_FLAG_PASSIVE,
+        .init = (void*) 0x00004003,
+        .animations = {
+        },
+    },
+    {
+        .id = NPC_FireBar_2C,
+        .settings = &N(NpcSettings_FireBar_Extra),
+        .pos = { 0.0f, 0.0f, 0.0f },
+        .yaw = 0,
+        .flags = NPC_FLAG_PASSIVE,
+        .init = (void*) 0x00004003,
```

Comment:
what does this do

Reply:
No idea yet. Only fire bars use this field in this way.